api/test/integration/test_logtest_endpoints.tavern.yaml

```diff
@@ -0,0 +1,164 @@
+            location: "master->/var/log/syslog"
+          codemsg: 1 # codemsg = 1 because there is a warning (token generated)
+      save:
+        json:
+          returned_token_1: data.token
+
+  - name: Run logtest with a valid body but no token
+    request:
+      verify: False
+      url: "{protocol:s}://{host:s}:{port:d}/logtest"
+      method: PUT
+      headers:
+        Authorization: "Bearer {test_login_token}"
+        content-type: application/json
+      json:
+        event: "Jun 24 11:54:19 Master systemd[2099]: Started VTE child process 20118 launched by terminator process 17756."
+        log_format: "syslog"
+        location: "master->/var/log/syslog"
+    response:
+      status_code: 200
+      json:
+        data:
+          token: !anystr
+          output:
+            full_log: "Jun 24 11:54:19 Master systemd[2099]: Started VTE child process 20118 launched by terminator process 17756."
+            location: "master->/var/log/syslog"
+          codemsg: 1 # codemsg = 1 because there is a warning (token generated)
+
+  - name: Run logtest with the token generated before
+    request:
+      verify: False
+      url: "{protocol:s}://{host:s}:{port:d}/logtest"
+      method: PUT
+      headers:
+        Authorization: "Bearer {test_login_token}"
+        content-type: application/json
+      json:
+        token: "{returned_token_1}"
+        event: "Jun 24 11:54:19 Master systemd[2099]: Started VTE child process 20118 launched by terminator process 17756."
+        log_format: "syslog"
+        location: "master->/var/log/syslog"
+    response:
+      status_code: 200
+      json:
+        data:
+          token: "{returned_token_1}"
+          output:
+            full_log: "Jun 24 11:54:19 Master systemd[2099]: Started VTE child process 20118 launched by terminator process 17756."
+            location: "master->/var/log/syslog"
+          codemsg: 0
+
+
+---
+test_name: DELETE /logtest/sessions
+
+stages:
+
+  - name: Run logtest in order to generate token
+    request:
+      verify: False
+      url: "{protocol:s}://{host:s}:{port:d}/logtest"
+      method: PUT
+      headers:
+        Authorization: "Bearer {test_login_token}"
+        content-type: application/json
+      json:
+        event: "Jun 24 11:54:19 Master systemd[2099]: Started VTE child process 20118 launched by terminator process 17756."
+        log_format: "syslog"
+        location: "master->/var/log/syslog"
+    response:
+      status_code: 200
+      json:
+        data:
+          token: !anystr
+          output:
+            full_log: "Jun 24 11:54:19 Master systemd[2099]: Started VTE child process 20118 launched by terminator process 17756."
+            location: "master->/var/log/syslog"
+          codemsg: 1 # codemsg = 1 because there is a warning (token generated)
+      save:
+        json:
+          returned_token_2: data.token
```

Comment:
Change the names of the tokens to make them more descriptive. The first one could be `returned_token` and this one could be `delete_session_token` for instance.